metabench.cmake

```diff
@@ -247,6 +247,11 @@ file(WRITE ${METABENCH_RB_PATH}
 "require 'time'                                                                                         \n"
 "                                                                                                       \n"
 "module Metabench                                                                                       \n"
+"  def self.median(values)                                                                              \n"
+"    values = values.sort                                                                               \n"
+"    return values[values.length / 2]                                                                   \n"
```

Comment:
When `values.length` is even, the median should be `(values[values.length / 2 - 1] + values[values.length / 2])/2`